docs/concepts/persistence.md

```diff
@@ -83,3 +82,20 @@ Some available pinning service providers are:
 - [Crust Network](https://crust.network/)
 
 See how to [work with remote pinning services](../how-to/work-with-pinning-services.md).
+
+## Long-term storage
+
+Storing data using a personal IPFS node is easy but inconvenient since you constantly have to manage your own hardware. This problem gave rise to pinning services that allow you to upload your data to a remotely-hosted IPFS node and retrieve it whenever you want. However, pinning services can be a dangerous form of long-term storage. Once a file, or collection of files, is no longer pinned, it may be lost entirely. There is no guarantee that the data is still accessible through IPFS. This is where [Filecoin](https://filecoin.io) comes in.
```

Comment:
I think this may come across a little too derogatory. 

Maybe change to: 

However, while paying pinning providers can be convenient - it still requires an individual to bear the cost of storing that data (you paying a pinning provider). If that one sponsor stops paying for that pinning, the content may be lost entirely. While IPFS guarantees that any content on the network is discoverable, it doesn't guarantee that any content is persistently available. This is where [Filecoin](https://filecoin.io) comes in.

Reply:
That's fair.

```suggestion
However, while paying pinning providers can be convenient it still requires an individual to bear the cost of storing that data. If that one sponsor stops paying for the service, the content may be lost entirely. While IPFS guarantees that any content on the network is discoverable, it doesn't guarantee that any content is persistently available. This is where [Filecoin](https://filecoin.io) comes in.
```

How's this @jnthnvctr?

Reply:
Oh, nevermind. Just seen Jessica's comment below.